documentation/create_overview_doc/README.md

```diff
@@ -51,21 +51,27 @@ sh
 kt
 ```
 
+For flow map generation with Code2Flow, the tool currently supports the following languages, each with specific dependencies:
+
+- JavaScript: Requires Acorn for parsing. Ensure Acorn is installed and accessible in your environment.
+- Ruby: Uses Parser for parsing. Ensure the Parser gem is installed and accessible.
+- PHP: Employs PHP-Parser for parsing. PHP-Parser should be installed and accessible in your environment.
+- Python: No additional dependencies are required for Python projects.
+
+If your project uses a language not supported by Code2Flow for flow map generation, the documentation generation capabilities will still be available, but without the flow map visualization for those specific languages.
+
 ## Prerequisites
 
 Ensure the following tools are installed:
 
 - `bito` : https://github.com/gitbito/CLI
 
-- `mermaidcli` : https://github.com/mermaid-js/mermaid-cli
+- `Code2Flow:` : https://github.com/scottrogowski/code2flow
```

Comment:
**Performance Issue**: The replacement of MermaidCLI with Code2Flow as a dependency for flow map generation might impact the performance if Code2Flow is not as optimized for certain use cases. <br> **Fix**: Benchmark the performance of Code2Flow against MermaidCLI for various sizes and complexities of diagrams to ensure that Code2Flow's performance is on par or better. Optimize Code2Flow usage or provide guidelines for its efficient use. <br> **Code Suggestion**: 
 ```
 - `Code2Flow:` : https://github.com/scottrogowski/code2flow
 + `Code2Flow:` : https://github.com/scottrogowski/code2flow Benchmark Code2Flow's performance against MermaidCLI and optimize usage based on findings.
 ```